packages/testing-library/src/index.js

```diff
@@ -0,0 +1,29 @@
+/**
```

Comment:
These files are copied from the source code of [RTL](https://github.com/testing-library/react-testing-library/blob/master/src/index.js). Should we include a copyright header for this?

Reply:
excuse my noob-ness :) why do we need to do this?

Reply:
Just a convention, I guess? I don't think that we _need_ to do this from legal stand point, but might still be required if we used to do this. I have no idea 😂